src/device-registry/services/nearest-device.js

```diff
@@ -0,0 +1,60 @@
+const { logElement } = require("../utils/log");
```

Comment:
Could we move this file to the utils and remove this services' folder from the repo? What could be the difference in the contents of `utils` and `services`? I am just thinking out loud. Otherwise, this is just a file organisation discussion.  Thoughts?